model/post_list.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 type PostList struct {
-	Order []string         `json:"order"`
-	Posts map[string]*Post `json:"posts"`
+	Order          []string         `json:"order"`
+	Posts          map[string]*Post `json:"posts"`
+	NextPostId     string           `json:"next_post_id,omitempty"`
```

Comment:
I think we might want to remove the `omitempty` here. I haven't thought a ton about it, but I think it might be useful on the client to be able to tell when there are no more posts (`next_post_id == ''`) vs not supported by the server (`next_post_id == undefined`)